subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarc.java

```diff
@@ -63,6 +67,7 @@
 
     public CodeNarc() {
         reports = getInstantiator().newInstance(CodeNarcReportsImpl.class, this);
+        compilationClasspath = new SimpleFileCollection();
```

Comment:
Right. Newing up a `SimpleFileCollection` did not feel right but I could not find the right way to create an empty file collection. Will keep `getProject.files()` in mind for the future.